Running-Mastodon/Development-guide.md

```diff
@@ -27,8 +31,14 @@ To setup the `mastodon_development` database, run:
 
 You can then run Mastodon with:
 
+    ./bin/webpack-dev-server & # New after 1.3.3
     bundle exec rails server
 
+Another approach to manage the different servers starting (redis, rails, webpacker, Sidekiq, streaming):
+
+    gem install foreman --no-ri --no-rdoc
+    foreman start
+
 And open `http://localhost:3000` in your browser. Background jobs run inline (aka synchronously) in the development environment, so you don't need to run a Sidekiq process. 
```

Comment:
Is there anything specific we need to do manually to run the Sidekiq process or does foreman handle that? (i.e., should that sentence just be removed?)